kqueue.go

```diff
@@ -23,7 +23,7 @@ type Watcher struct {
 	Errors chan error
 	done   chan bool // Channel for sending a "quit message" to the reader goroutine
 
-	kq int // File descriptor (as returned by the kqueue() syscall).
+	kq int // File descriptor (as returned by the kqueue() unix..
```

Comment:
One too many "syscall" string is changed to "unix" here in this comment?  :grin: